<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostProcessIR</class>
 <widget class="QDialog" name="PostProcessIR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>411</x>
     <y>40</y>
     <width>771</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="dispLayout"/>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>17</height>
    </size>
   </property>
   <property name="text">
    <string>File Selected: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dispSelectedFile">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>1000</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>601</width>
     <height>17</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>17</height>
    </size>
   </property>
   <property name="text">
    <string>Use Select File Button</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>680</y>
     <width>771</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSlider" name="sl">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>739</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="slStartF">
        <property name="text">
         <string>Please Select File</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slMidF">
        <property name="text">
         <string>Please Select File</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slEndF">
        <property name="text">
         <string>Please Select File</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="slStartT">
        <property name="text">
         <string>Please Select File</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slMidT">
        <property name="text">
         <string>Please Select File</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slEndT">
        <property name="text">
         <string>Please Select File</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="selectFileBut">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Select HDF5 File</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="cmIronBut">
          <property name="text">
           <string>Ironblack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmGrayBut">
          <property name="text">
           <string>Grayscale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmRainBut">
          <property name="text">
           <string>Rainbow</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="playVidBut">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Play Video</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pauseVidBut">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Pause Video</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="rotButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Rotate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextFrame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Next Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="prevFrame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Previous Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Start Frame:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Stop Frame:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="startEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="stopEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="saveAsVideoSS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save AVI Video from Start to Stop Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tempScaleBut">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Generate Figure with Temp Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveCvImageBut">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save PNG Image (Without Axes)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="makeTiffBut">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Generate Tiff File (Full Duration)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="displayC">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Display Celsius</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="displayF">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Display Fahrenheit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="cursorTempLabel">
        <property name="text">
         <string>Cursor Temp:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxTempLabel">
        <property name="text">
         <string>Current Max Temp:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxTempLocLabel">
        <property name="text">
         <string>Max Temp Loc:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minTempLabel">
        <property name="text">
         <string>Current Min Temp: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minTempLocLabel">
        <property name="text">
         <string>Min Temp Loc:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentFrameDisp">
        <property name="text">
         <string>Current Frame:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentTimeLabel">
        <property name="text">
         <string>Current Time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Command Log:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="history"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
